tests/kernel/futex/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_ZTEST=y
+CONFIG_IRQ_OFFLOAD=y
+CONFIG_TEST_USERSPACE=y
+CONFIG_SMP=n
```

Comment:
why was this specifically disabled?

Reply:
the cases so far don't consider SMP.

Reply:
Remove this line.
We have no SMP enabled userspace targets, and when we do we will need to ensure this works.